evap/staff/templates/staff_evaluation_evaluation_period.html

```diff
@@ -1,18 +1,19 @@
 {% if evaluation.is_single_result %}
-    <span class="small">
-        {{ evaluation.vote_start_datetime|date:"Y-m-d" }}
-    </span>
+    <span class="small">{{ evaluation.vote_start_datetime|date:"Y-m-d" }}</span>
 {% else %}
     <span class="small">
-        {{ evaluation.vote_start_datetime }}{% if not start_only %} &ndash;<br />
-        {{ evaluation.vote_end_date }} ({% blocktrans count runtime=evaluation.runtime %}{{ runtime }} day{% plural %}{{ runtime }} days{% endblocktrans %}){% endif %}
-    </span><br />
+        {{ evaluation.vote_start_datetime }}
+        {% if not start_only %}
+            &ndash;
```

Comment:
If I understand this correctly, this dash lost the space before it by this change. There are more instances of this.